internal/inventory/asset.go

```diff
@@ -158,6 +158,11 @@ type AssetEvent struct {
 	Labels        map[string]string
 	Tags          []string
 	RawAttributes *any
+	Related       *Related
+}
+
+type Related struct {
+	Id []string `json:"id,omitempty"`
```

Comment:
in order to test `WithRelatedAssetIds` i needed an exported field, so i replaced `Entity.relatedEntityId` with `Related.Id`